### DISCUSSION TITLE
Live connection latency metrics panel

### DISCUSSION THREAD

**Discussion by banisadr** [foxglove/community#414] [Q&A]
Panel that displays per-topic latency metrics for monitoring and debugging the state of a live robot connection.

**Comment by jhurliman** [comment]
What about the connection sidebar? That's where we currently show topic message counts and frequency.

**Comment by jtbandes** [comment]
We also have the Data Source Info panel (which is largely the same as the sidebar): https://foxglove.dev/docs/studio/panels/data-source-info
